undefined: setup for GOARCH=wasm GOOS=js

It is impossible to use any `google/logger` based package (e.g. https://github.com/google/go-sev-guest) when compiling to WASM, since `google/logger` does not provide a default `setup` function:

```
GOARCH=wasm GOOS=js go run -exec="/usr/local/go/misc/wasm/go_js_wasm_exec" ./examples/client
# github.com/google/logger
/home/sza-1/go/pkg/mod/github.com/google/logger@v1.1.1/logger.go:81:27: undefined: setup
```

--- a/go.mod
+++ b/go.mod
@@ -1,5 +1,5 @@
 module github.com/google/logger
 
-go 1.12
+go 1.18
 
 require golang.org/x/sys v0.1.0

--- a/logger_other.go
+++ b/logger_other.go
@@ -1,0 +1,12 @@
+//go:build !unix && !windows
+
+package logger
+
+import (
+	"errors"
+	"io"
+)
+
+func setup(src string) (io.Writer, io.Writer, io.Writer, error) {
+	return nil, nil, nil, errors.New("system logging not implemented")
+}

--- a/logger_syslog.go
+++ b/logger_syslog.go
@@ -1,4 +1,4 @@
-// +build linux darwin freebsd
+//go:build unix
 
 /*
 Copyright 2016 Google Inc. All Rights Reserved.

--- a/logger_windows.go
+++ b/logger_windows.go
@@ -1,3 +1,4 @@
+//go:build windows
 /*
 Copyright 2016 Google Inc. All Rights Reserved.
 Licensed under the Apache License, Version 2.0 (the "License");

